CodeStarterCamp_Week1/main.swift

```diff
@@ -7,6 +7,21 @@
 //
 
 import Foundation
+import Swift
 
-print("Hello, World!")
+//단 수를 입력하면 아이스크림 부분을 그리는 함수
+func drawIcecream(dan: Int){
+    for count in 1...dan{
+        print("***********")
+    }
+}
```

Comment:
네이밍에 카멜케이스를 잘 적용해주셨네요 👍
파라미터의 이름은 한글 발음을 적는 `dan` 보다는 일관성을 위해 영어 단어를 사용하면 좋을 것 같습니다.

또한, 일반적으로 사용하는 중괄호 컨벤션은 아래와 같습니다.
```suggestion
func drawIcecream(dan: Int) {
    for count in 1...dan {
        print("***********")
    }
}
```